pkg/rancher-desktop/pages/snapshots/create.vue

```diff
@@ -55,8 +57,11 @@ export default Vue.extend<Data, Methods, Computed, never>({
   },
 
   methods: {
-    goBack() {
-      this.$router.push({ name: 'Snapshots' });
+    goBack(event: SnapshotEvent) {
+      this.$router.push({
+        name:   'Snapshots',
+        params: { event } as any,
```

Comment:
`params` has a type of `Dictionary<string>`. While the code works as written, I think it would be safer if we destructured `event`. This would fit better, as now our route parameters are key-value pairs where both keys and values are strings.

```suggestion
        params: { ...event },
```

Keep in mind that doing this will break the implementation in `pkg/rancher-desktop/components/Snapshots.vue`